### PR TITLE
Added adocs for try section

### PR DESCRIPTION
**Pull request by Ahmad-Ayyoub**

### CODE REVIEW

**Comment by yeekangc** [comment]
I don't see the changes for the feedback from @Kubik42 above?

Also, any suggestion for names of the files?  "tryitoutintro", "tryitoutfinish"?  Or, "tryoutintro", "tryoutend"?

File: template.adoc [hunk 1]
```diff
@@ -58,7 +58,9 @@ include::{common-includes}/gitclone.adoc[]
 // they will be building.
 === Try what you'll build
 
+include::{common-includes}/tryintro.adoc[]
```
Comment (evelinec) [inline template.adoc]: In between here, perhaps you want to add a comment for guide writer to add the talk about what user are expected to see. ie, // Describe what user are expected to see

**Comment by Ahmad-Ayyoub** [comment]
@yeekangc me and @Kubik42 discussed and we only had to remove the duplicate files. Because the "application" word is used for maven goals like install , liberty:start-server etc. . So we can not remove that. Otherwise we will stay something like build the guide (should be application not guide).

@evelinec will do.

**Comment by Ahmad-Ayyoub** [comment]
@yeekangc & @evelinec please take a look

File: template.adoc (hunk 1 above)
Comment (Ahmad-Ayyoub) [reply template.adoc]: Fixed

File: template.adoc [hunk 2]
```diff
@@ -58,7 +58,11 @@ include::{common-includes}/gitclone.adoc[]
...
```
Comment (evelinec) [inline template.adoc]: Looks like the adoc file names in the "include" links need to be updated.

**Comment by evelinec** [comment]
Others look good.

File: template.adoc (hunk 2 above)
Comment (Ahmad-Ayyoub) [reply template.adoc]: Fixed

**Comment by yeekangc** [comment]
We should consider more obvious filenames like trywhatyoubuild-intro and trywhatyoubuild-end.

File: tryoutintro.adoc [hunk 3]
```diff
@@ -0,0 +1,11 @@
+=== Try what you'll build
```
[the remark this answers is not in the document]
Comment (Ahmad-Ayyoub) [reply tryoutintro.adoc]: Removed it from template

**Comment by yeekangc** [comment]
Yes, there were suggestions only.  We should see if there are better ones and go with what's best.

**Comment by Ahmad-Ayyoub** [comment]
Ok I will update that then.

**Comment by Ahmad-Ayyoub** [comment]
Renamed

**Comment by Ahmad-Ayyoub** [comment]
@yeekangc can you approve?

File: trywhatyoubuild-intro.adoc [hunk 4]
```diff
@@ -0,0 +1,11 @@
+=== Try what you'll build
+
+The `finish` directory in the root of this guide contains the finished application. Give it a try before you proceed.
+
+To try out the application, first navigate to the `finish` directory and then run the following
+Maven goals to build the application and run it inside Open Liberty:
```
Comment (yeekangc) [inline trywhatyoubuild-intro.adoc]: It should be "goal" and not "goals"?

Comment (Ahmad-Ayyoub) [reply trywhatyoubuild-intro.adoc]: Fixed

**Pull request merged**